.github/workflows/integration_tests.yml

```diff
@@ -137,6 +138,17 @@ jobs:
 
       - name: Submit to IQM Demo server
         if: (success() || failure()) && (inputs.target == 'iqm' || github.event_name == 'schedule' || inputs.target == 'nightly')
+        # The following tests are disabled because the demo server only supports
+        # 5 qubits and the test tries to use more.
+        # - test/NVQPP/integration/graph_coloring-1.cpp
+        # - test/NVQPP/integration/graph_coloring.cpp
+        # The following tests are disabled because a mapping algorithm is needed
+        # to account for qubit connection limitations (PR #610 in work).
```

Comment:
nit: I generally prefer links to issues instead of mentioning specific PRs.

Reply:
Fixed in [ba40e61](https://github.com/NVIDIA/cuda-quantum/pull/638/commits/ba40e612b8da402b1969eb4c991b4614f5319cb0)